src/ScalarAuthClient.ts

```diff
@@ -31,6 +30,12 @@ const imApiVersion = "1.1";
 // TODO: Generify the name of this class and all components within - it's not just for Scalar.
 
 export default class ScalarAuthClient {
+    private apiUrl: string;
+    private uiUrl: string;
+    private scalarToken: string;
+    private termsInteractionCallback: TermsInteractionCallback;
+    private isDefaultManager: boolean;
+
     constructor(apiUrl, uiUrl) {
         this.apiUrl = apiUrl;
         this.uiUrl = uiUrl;
```

Comment:
this could be `constructor(private apiUrl: string, private uiUrl: string) {` and then lines 33, 34 could be removed

Reply:
Ah cool, hadn't seen that syntax yet, will change.